13 ShouldUpdate/readme.md

```diff
@@ -194,6 +194,29 @@ for that in _face.tsx_
     );
   }
   ```
+- Let's create a simple utility file under _src_ named _utils.ts_:
+
+  ```ts
+  const ranges = [100, 200, 300, 400];
+
+  export function isThereSomeRangeTransition (currentLevel, nextLevel) {
+    return ranges.some((range) => {
+      return isAnUpperRangeTransition(currentLevel, nextLevel, range) ||
+        isALowerRangeTransition(currentLevel, nextLevel, range);
+    });
+  }
```

Comment:
Sorry for coming so late to this pull, just realized today when reviewing new ones. 

@nasdan found another way to implement this reusing the calculateSatisfaction class, what do you think?

```javascript
export const calculateSatisfactionClass = (level: number) => {
  if (level < 100) {
    return "very-dissatisfied"
  }

  if (level < 200) {
    return "somewhat-dissatisfied"
  }

  if (level < 300) {
    return "neither"
  }

  if (level < 400) {
    return "somewhat-satisfied"
  }

  return "very-satisfied"
}

export const isRangeChange = (oldValue : number, newValue : number) => {
  const oldValueClass = calculateSatisfactionClass(oldValue); 
  const newValueClass = calculateSatisfactionClass(newValue);
  return oldValueClass !== newValueClass; 
}
```

If you like this way, one thing that could be enhanced is the _calculateSatisfactionClass_ method.

Are you keen on updating the pull? I will make a quick check an merge.

Thanks